src/containers/CreateTripModal/Modal.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import "./Modal.css";
+
+const Modal = props => {
+    const [show, showmodal] = useState(false);
+
+    const handleClose = () => showmodal(false);
+
+    const handleShow = () => showmodal(true);
+
+    return (
+        <>
+            <button onClick={handleShow}>Create Trip</button>
+
+            <div className={show ? "displayModal flipOutY" : "displayNone"} >
+                <div className="displayModalMain">
+                    <button className="modalCloseButton" onClick={handleClose}>x</button>
+                    <div className="modalContent">
+                        {props.modalContent}
```

Comment:
It's a standard practice to use `props.children` rather than some other arbitrarily named property. That way you can use it like this:

```js
<Modal>
    <p>(some modal content)</p>
</Modal>
```